www/components/shelf/shelf.js

```diff
@@ -15,8 +15,6 @@ class shelf extends LitElement {
 
   constructor() {
     super();
-    this.selectedIndex = 0;
```

Comment:
would still want to keep a default value, or is that part of the bug?  I see below we set it to empty string
```
if (this.selectedIndex === previousSelected) {
  this.selectedIndex = '';
}
```

Reply:
Yea it's redundant apparently

Reply:
My comment was more towards keep the initialization though, for `this.selectedIndex`.  Did it need to be removed to fix the bug?  Otherwise, it's generally best practice to create / initialize your members.